dmt/settings_shared.py

```diff
@@ -81,7 +81,7 @@
     messages.ERROR: 'danger'
 }
 
-SERVER_EMAIL = 'pmt@ccnmtl.columbia.edu'
+SERVER_EMAIL = 'pmt@ctl.columbia.edu'
```

Comment:
Does the SES gateway whitelist ctl.columbia.edu? Worth a check.

Reply:
Good catch, it looks like we only have the `ccnmtl.columbia.edu` "Domain Identity" set up on SES. I'm not so familiar with SES right now, so we'll have to investigate further.